eth-connector/src/connector_impl.rs

```diff
@@ -1,24 +1,8 @@
-use crate::{
```

Comment:
This file looks strange now. It only contains errors. I think it should be restructured.